manifests/config.pp

```diff
@@ -415,4 +435,15 @@
     require => File['conntrackd-confdir'],
     notify  => Service['conntrackd'],
   }
+
+  if $systemd {
```

Comment:
Any modification to this file should imply a `systemctl daemon-reload`.
This can be done, if the module of systemd is set as dependency in this conntrackd module 

Here is the resource of systemd: 
https://github.com/camptocamp/puppet-systemd/blob/1.1.1/manifests/systemctl/daemon_reload.pp

Maybe this systemd unit could be written too by this module